nixos/modules/services/web-apps/pgpkeyserver-lite.nix

```diff
@@ -0,0 +1,76 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pgpkeyserver-lite;
+  sksCfg = config.services.sks;
+
+  webPkg = cfg.package;
+
+in
+
+{
+
+  options = {
+
+    services.pgpkeyserver-lite = {
+
+      enable = mkEnableOption "pgpkeyserver-lite";
```

Comment:
It would probably make sense to mention here that a nginx vhost will be created. It is an implementation detail but an important one if the user decides to run another http daemon on the same machine.